tests/test_load_archival.py

```diff
@@ -61,6 +63,9 @@ def test_load_directory(metadata_storage_connector, passage_storage_connector, c
     elif passage_storage_connector == "chroma":
         print("testing chroma passage storage")
         # nothing to do (should be config defaults)
+    elif passage_storage_connector == "qdrant":
+        print("Testing Qdrant passage storage")
+        config.archival_storage_uri = "localhost:6333"
```

Comment:
Can we setup a docker container to run the DB service for the tests, like we do with postgres? Otherwise this can't pass the github workflow tests.

Reply:
Qdrant tests skipped if a server wasn't running.
https://github.com/cpacker/MemGPT/blob/f4349aa5cdc288ad237f6c466cac74b9f2455d24/tests/utils.py#L129-L148

Reply:
I've added a step to run a Qdrant container in the CI as requested.
https://github.com/cpacker/MemGPT/pull/1023/commits/f4349aa5cdc288ad237f6c466cac74b9f2455d24